scripts/leapfrog/playbooks/remove-old-agents-from-maas.yml

```diff
@@ -2,8 +2,13 @@
 - hosts: hosts
   gather_facts: no
   tasks:
-    - shell: "rm -f {% raw %}/etc/rackspace-monitoring-agent.conf.d/*${{% endraw %}{{ item }}{% raw %}}*{% endraw %}"
-      args:
-        executable: /bin/bash
-      with_file:
-        - "/etc/openstack_deploy/leapfrog_remove_remaining_old_containers"
+    - name: Get list of old containers
```

Comment:
I'd rather keep what's on left, replacing the shell line like this:

``"rm /etc/rackspace-monitoring-agent.conf.d/*{{ item }}*"``